frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -366,4 +385,13 @@ export default {
 .e-category-chip-save-icon {
   font-size: 18px;
 }
+
+.ec-schedule-entry {
+  transition: max-width 0.7s ease;
+}
+
+.resize-icon,
+.resize-icon ::v-deep svg {
+  width: 28px !important;
```

Comment:
In which case do you need the !important?
I tried it in the dev console and did not need the !important.

And: https://stackoverflow.com/a/55368933/1864100

Reply:
Yes you do :D 
Otherwise the icon is 24px wide. As I've created an 24×28px wide icon, which I want 28px wide, I need to overwrite Vuetify.

I'm not sure what you want to tell me with this link? v-deep as a combinator selector is still valid. Only recently from v2.7 on it is deprecated. Could you please create an issue to tackle all remaining ::v-deep that are not pseudo-functions.

Reply:
As i interpret it, you selector is more specific than the selector from vuetify, thus the important is not needed.

![grafik](https://github.com/ecamp/ecamp3/assets/1506818/d538da4c-c219-43cc-a66e-bb8dfb213222)

Reply:
Maybe this is correct for the `<svg>`, but vuetify also specifies the width on the `span.v-icon` again. For me in Chrome your solution does not work.